### PR TITLE
CompatabilityPlugin typings added

### PR DESCRIPTION
**Pull request by Legends**
**What kind of change does this PR introduce?**
CompatibilityPlugin typings added

**Did you add tests for your changes?**
no

**Does this PR introduce a breaking change?**
no

**What needs to be documented once your changes are merged?**
no

### CODE REVIEW

File: lib/CompatibilityPlugin.js [hunk 1]
```diff
@@ -9,6 +9,11 @@ const ConstDependency = require("./dependencies/ConstDependency");
 const NullFactory = require("./NullFactory");
 
 class CompatibilityPlugin {
+	/**
+	 * Apply the plugin
+	 * @param {TODO} compiler Webpack Compiler
```
Comment (sokra) [inline lib/CompatibilityPlugin.js]: TODO => Compiler

Comment (Legends) [reply lib/CompatibilityPlugin.js]: I've copied the typing from [mohsens1 typing for DefinePlugin].(https://github.com/webpack/webpack/pull/7356/commits/800e0de25b8e0e0fc9ae69b9e76dbec00fb5895f).
I thought, he has his reasons not to use `Compiler` here.
Np, I will change it.

Comment (Legends) [reply lib/CompatibilityPlugin.js]: done

**Comment by sokra** [comment]
Thanks

**Pull request merged**